cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -271,31 +282,40 @@ static void sc_populate_libgl_with_hostfs_symlinks(const char *libgl_dir, const
     }
 }
 
+static void sc_mkdir_and_mount_tpmfs(const char *dir) {
+    if (sc_ensure_mkdir(dir, 0755, 0, 0) != 0) {
+        die("cannot create tmpfs target %s", dir);
+    }
+
+    debug("mounting tmpfs at %s", dir);
+    if (mount("none", dir, "tmpfs", MS_NODEV | MS_NOEXEC, NULL) != 0) {
```

Comment:
We have some helpers that protect against mounting through symlinks. Shall we use those here and in other helpers?